app/src/main/java/com/sopt/now/LoginActivity.kt

```diff
@@ -0,0 +1,71 @@
+package com.sopt.now
+
+import UserData
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.enableEdgeToEdge
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.ViewCompat
+import androidx.core.view.WindowInsetsCompat
+import com.sopt.now.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+    private val users : MutableList<UserData> = mutableListOf()
+
+    private val resultLauncher = registerForActivityResult(
+        ActivityResultContracts.StartActivityForResult()
+    ){
+        if(it.resultCode == RESULT_OK){
```

Comment:
```suggestion
    ){ activityResult -> 
        if(activityResult.resultCode == RESULT_OK){
```
가독성을 위해서 이런 식으로 it 값을 명시해주는 게 좋아요 ~